crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -108,6 +108,22 @@ impl Mesh {
         self.attributes.get_mut(&name.into())
     }
 
+    /// Returns an iterator that yields references to the data of each vertex attribute.
+    pub fn attributes(&self) -> impl Iterator<Item = (&'_ str, &'_ VertexAttributeValues)> + '_ {
+        self.attributes
+            .iter()
+            .map(|(name, values)| (name.as_ref(), values))
+    }
+
+    /// Returns an iterator that yields mutable references to the data of each vertex attribute.
+    pub fn attributes_mut(
+        &mut self,
+    ) -> impl Iterator<Item = (&'_ str, &'_ mut VertexAttributeValues)> {
```

Comment:
If possible, I would prefer no lifetimes here at all. If they're required, we should name them though.